src/internal/header.rs

```diff
@@ -31,13 +36,15 @@ impl Header {
                 self.0,
                 tag as i32,
                 &mut td,
-                librpm_sys::headerGetFlags_e_HEADERGET_MINMEM,
+                librpm_sys::headerGetFlags_e_HEADERGET_ALLOC,
             )
         };
 
-        assert_ne!(rc, 0, "headerGet returned non-zero status: {}", rc);
```

Comment:
It should be a non-fatal error to fetch a tag that doesn't exist - epoch is one such tag.